Code/Source/solver/read_files.cpp

```diff
@@ -3019,6 +3038,152 @@ void read_wall_props_ff(ComMod& com_mod, const std::string& file_name, const int
   }
 }
 
+//--------------------
+// Adds a BC to lEq.bc at the end for a cap. Copies most of the BC
+// info from lEq.bc[iBc], which corresponds to the capped surface.
+// Also, sets info about capping face in capped face (capName and
+// capID fields).
+//
+void addCapBC(ComMod& com_mod, eqType& lEq, int iBc) {
+  // We are adding a new BC for the cap, so we need to update 
```

Comment:
This function should be renamed to be snake case and should have a better name, perhaps `add_coupling_cap_bc`.